recipes/libgit2/all/conanfile.py

```diff
@@ -151,6 +151,8 @@ def generate(self):
         tc.variables["REGEX_BACKEND"] = self.options.with_regex
         if is_msvc(self):
             tc.variables["STATIC_CRT"] = is_msvc_static_runtime(self)
+        if Version(self.version) >= "1.8.0":
+            tc.variables["USE_SSH"] = "libssh2"
```

Comment:
This keeps the same old behaviour as per https://github.com/libgit2/libgit2/compare/v1.7.2...v1.8.0#diff-95a55896bed1a5af65001605d4d237f52bed5366bd33d43528638dc756e26efbR7, thanks!